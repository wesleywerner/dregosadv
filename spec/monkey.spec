superbrief
gonear clearing portal
read compass
show compass to monkey
talk to monkey
say hi to monkey
climb tree
down

take compass
ask monkey for compass
wait
wait
wait
wait
whip the sky
gonear garden portal
whip the tree
take banana
gonear clearing portal
show the banana to the monkey
throw banana at monkey
whip monkey
drop the banana
wait
wait
take compass
