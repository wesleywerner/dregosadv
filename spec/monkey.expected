   The Maro River                                 Time:  3:00 pm
  
  Dr Ego and the Egg of Man-Toomba
  An Interactive Fiction Story
  by Special Agent.
  Release 1 / Serial number 200410 / Inform v6.33 Library v6.12.3pre SX
  
  The Maro River (in the canoe)
  You are in a canoe on the Maro river. Your guide is with you, rowing the
  craft.
. 
> >   The Maro River                                 Time:  3:00 pm
. 
  Dr Ego and the Egg of Man-Toomba is now in its "superbrief" mode, which
  always gives short descriptions of locations (even if you haven't been
  there before).
  
> >   In a Clearing                                  Time:  3:00 pm
. 
  In a Clearing
  
> >   In a Clearing                                  Time:  3:01 pm
. 
  The needle points towards north.
  The monkey seems very interested in the device.
  
> >   In a Clearing                                  Time:  3:02 pm
. 
  It seems curious about the compass.
  
> >   In a Clearing                                  Time:  3:03 pm
. 
  To ask the monkey something type "ask the monkey about (thing or topic)".
  To say something to the monkey type "say (noun or topic)".
  
> >   In a Clearing                                  Time:  3:04 pm
. 
  "*Grunts*"
  
> >   Top of Tree                                    Time:  3:05 pm
. 
  You climb up the strong branches of the fig tree.
  
  Top of Tree
  
  The monkey is not impressed, it climbs down the tree.
  
> >  As you descend you drop your compass to the ground below. CLINK!
  
   --- more ---
)    In a Clearing                                  Time:  3:06 pm
. 
  In a Clearing
  
  There is a monkey here. It has your compass.
  
> >   In a Clearing                                  Time:  3:07 pm
. 
  When you approach the small critter it deftly moves out of your reach.
  "*Pants*Grunt*"
  
  The monkey scrambles up the fig tree.
  
> >   In a Clearing                                  Time:  3:08 pm
. 
  "*Grunts*" it replies, gripping the device tightly.
  
> >   In a Clearing                                  Time:  3:09 pm
. 
  Time passes.
  
> >   In a Clearing                                  Time:  3:10 pm
. 
  Time passes.
  
> >   In a Clearing                                  Time:  3:11 pm
. 
  Time passes.
  
> >   In a Clearing                                  Time:  3:12 pm
. 
  Time passes.
  
  The monkey climbs down from the fig tree.
  
> >   In a Clearing                                  Time:  3:13 pm
. 
  (the sky)
  You crack the whip at the sky -- *SNAP*
  
  The piercing sound startles the monkey, it launches up into the air and
  scrambles up the fig tree.
  
> >   Garden                                         Time:  3:13 pm
. 
  Garden
  
  On the fruit trees is a banana.
  
> >   Garden                                         Time:  3:14 pm
. 
  You take careful aim at the bunch of bananas up in the tree, swing your arm
  in a smooth arch and *SNAP* -- a single banana drops to the ground.
  
> >   Garden                                         Time:  3:15 pm
. 
  Taken.
  
> >   In a Clearing                                  Time:  3:15 pm
. 
  In a Clearing
  
> >   In a Clearing                                  Time:  3:16 pm
. 
  The small monkey head swivels side-to-side as you wave the banana, showing
  it off. The monkey likes what it sees but looks hesitant to go near you.
  
> >   In a Clearing                                  Time:  3:17 pm
. 
  Maybe it would be better to just drop the banana.
  
> >   In a Clearing                                  Time:  3:18 pm
. 
  You are not fond of hurting other living things.
  
  The monkey climbs down from the fig tree.
  
> >   In a Clearing                                  Time:  3:19 pm
. 
  Dropped.
  
  The monkey stares at the banana, then glares at you, looks at the banana
  again, smacking it's lips.
  
> >   In a Clearing                                  Time:  3:20 pm
. 
  Time passes.
  
  The monkey edges closer to the banana while keeping one eye on you.
  
> >   In a Clearing                                  Time:  3:21 pm
. 
  Time passes.
  
  The monkey grabs the banana, stuffing its face with the delicious fruit. It
  seems distracted and left the compass lying on the ground.
  
> >   In a Clearing                                  Time:  3:22 pm
. 
  Taken.
  
  The monkey picks something off the ground, and eats it.
  
> >